patients/get_patient_info.php

```diff
@@ -71,112 +90,112 @@
           ." FROM ".TBL_PAT_ACC_ON
           ." WHERE ".COL_POR_USER."=?";
             $auth = privQuery($sql, array($_POST['uname']));
-    if($auth===false)
+if($auth===false)
+{
+    session_destroy();
+    header('Location: '.$landingpage.'&w');
+    exit;
```

Comment:
As best as I can tell, this isn't duplicated. The diff on this file is just poorly aligned.